src/Effects/CombFilterBank.cpp

```diff
@@ -103,24 +103,26 @@ namespace zyn {
             for (unsigned int j = 0; j < nrOfStrings; ++j)
             {
                 // calculate the feedback sample positions in the buffer
-                const float pos = float(mem_size-buffersize+i) - delays[j];
+                const float pos_reader = fmod(float(pos_writer) - delays[j], mem_size);
```

Comment:
what's wrong?

Reply:
Answer: Other than the integer mod operator (%) std::fmod returns negative result if first parameter is negative.

Reply:
Is there any specific reason we are using float for offsets and not ssize_t ?

Reply:
Or for that sake int or unsigned ? Buffers are usually not that big.

Reply:
I see. We are interpolating.

Reply:
const float pos_reader = fmod(mem_size + float(pos_writer) - delays[j], mem_size);

Reply:
thx